{"uuid": "d4d96ea3-6a06-420b-af43-3e2c1e6bd8c6", "children": ["934d0b7d-03be-4dea-ba49-ca9f0fa06c30", "782ff460-a30d-43fc-b793-a7f8e67180d1", "cae54483-1599-43f1-aa85-27af5ee8975d", "fb980587-ef96-493a-8e78-5d9ceb8c6436", "2daf0e15-2514-4eae-a960-74a8e17768a7", "545238e6-b00e-4f6b-8ae2-89271ee8ed4b", "e09adeb4-1efd-4dff-a2f0-226d175ee217"], "befores": [{"name": "setup_browser", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: <html>\r\n<head><title>401 Authorization Required</title></head>\r\n<body bgcolor=\"white\">\r\n<center><h1>401 Authorization Required</h1></center>\r\n<hr><center>nginx/1.14.2</center>\r\n</body>\r\n</html>\r\n\n\n", "trace": "  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/venv/lib/python3.11/site-packages/pluggy/_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 1130, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/venv/lib/python3.11/site-packages/_pytest/fixtures.py\", line 902, in call_fixture_func\n    fixture_result = next(generator)\n                     ^^^^^^^^^^^^^^^\n  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/conftest.py\", line 42, in setup_browser\n    driver = webdriver.Remote(\n             ^^^^^^^^^^^^^^^^^\n  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/venv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 286, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/venv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 378, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/venv/lib/python3.11/site-packages/selenium/webdriver/remote/webdriver.py\", line 440, in execute\n    self.error_handler.check_response(response)\n  File \"/Users/lasha/PycharmProjects/Jenkins_Lasha/venv/lib/python3.11/site-packages/selenium/webdriver/remote/errorhandler.py\", line 209, in check_response\n    raise exception_class(value)\n"}, "start": 1685198196315, "stop": 1685198196530}], "start": 1685198196315, "stop": 1685198196660}